pkgs/applications/misc/houdini/runtime.nix

```diff
@@ -1,11 +1,11 @@
-{ lib, stdenv, requireFile, callPackage}:
+{ requireFile, callPackage}:
 
 callPackage ./runtime-build.nix rec {
-  version = "20.0.688";
+  version = "20.5.278";
   eulaDate = "2021-10-13";
-  src = requireFile rec {
+  src = requireFile {
     name = "houdini-${version}-linux_x86_64_gcc11.2.tar.gz";
-    sha256 = "99f9088824c328de9d351f037f26ff1cba960fbd9b4e2ed1d52601680d3512a6";
+    sha256 = "3b9a00a220e2bbec6b9837d39e294fa99b9ef3e7575531e72b4cab5fa9a84d27";
```

Comment:
```suggestion
    hash = "3b9a00a220e2bbec6b9837d39e294fa99b9ef3e7575531e72b4cab5fa9a84d27";
```

Reply:
doesn't this require appending `sha256-` to the hash itself?

Reply:
ok, done, had to convert hash to sri form

Reply:
yes. and converting it. It should be `sha256-O5oAoiDiu+xrmDfTnilPqZue8+dXVTHnK0yrX6moTSc=` (got with `nix-hash --to-sri --type sha256 3b9a00a220e2bbec6b9837d39e294fa99b9ef3e7575531e72b4cab5fa9a84d27`).
I don't have `houdini` locally so you can get the hash with `nix-prefech-url`